Microsoft.AspNetCore.Components.Server/MessagePack/BufferWriter.cs

```diff
@@ -0,0 +1,166 @@
+using System;
+using System.Buffers;
+using System.Runtime.CompilerServices;
+
+namespace MessagePack
+{
+	/// <summary>
+	/// A fast access struct that wraps <see cref="T:System.Buffers.IBufferWriter`1" />.
+	/// </summary>
+	internal ref struct BufferWriter
```

Comment:
Do we still have this? I can't find it in our sources ATM.

Reply:
This might be from the submodule we compile in to Components.Server

Reply:
Yeah, that's what I eventually figured. I don't think we need to API review in that case....